packages/graphiql-react/src/explorer/components/doc-explorer.tsx

```diff
@@ -69,37 +63,29 @@ export function DocExplorer(props: DocExplorerProps) {
 
   return (
     <section
-      className="doc-explorer"
-      key={navItem.name}
+      className="graphiql-doc-explorer"
       aria-label="Documentation Explorer">
-      <div className="doc-explorer-title-bar">
-        {prevName && (
-          <button
-            className="doc-explorer-back"
-            onClick={pop}
-            aria-label={`Go back to ${prevName}`}>
-            {prevName}
-          </button>
-        )}
-        <div className="doc-explorer-title">
-          {navItem.title || navItem.name}
+      <div className="graphiql-doc-explorer-header">
+        <div className="graphiql-doc-explorer-header-content">
+          {prevName && (
+            <a
+              href="#"
+              className="graphiql-doc-explorer-back"
+              onClick={pop}
+              aria-label={`Go back to ${prevName}`}>
+              <ChevronLeftIcon />
+              {prevName}
+            </a>
+          )}
+          <div className="graphiql-doc-explorer-title">
+            {navItem.title || navItem.name}
+          </div>
```

Comment:
The `<a href="#"` is causing me trouble in an application where I embed the DocExplorer in a react app using the HashRouter. Clicking these back-buttons now navigates me to the index page of my whole application because it navigates to `#`. Do you have any advice for me how to work around this?

Reply:
Hey @Felk, thanks for calling this our! Can you please open a separate issue for this? It's hard to properly track this in comments of an already merged PR.

Reply:
Hey, I can't reproduce the issue with the newest live demo and it looks like it should have been fixed in https://github.com/graphql/graphiql/pull/2905

But that was already released in [@graphiql/react@0.15.0](https://github.com/graphql/graphiql/releases/tag/%40graphiql%2Freact%400.15.0). I'll have to check if I just used an old version 😬 if you don't hear from me, all is well. Otherwise I'll open a new issue!